doc/nrf/ug_nrf9160.rst

```diff
@@ -251,8 +251,10 @@ To perform a FOTA upgrade, complete the following steps:
       * If you want to perform a full modem firmware upgrade, an external flash memory with minimum 2MB is required.
 
 #. Create a binary file that contains the new image.
-      This step does not apply for upgrades of the modem firmware.
-      You can download delta patches and full modem firmware upgrade binaries for the modem firmware from the `nRF9160 product website (compatible downloads)`_.
+
+      .. note::
+         This step does not apply for upgrades of the modem firmware.
+         You can download delta patches and full modem firmware upgrade binaries for the modem firmware from the `nRF9160 product website (compatible downloads)`_.
```

Comment:
"full modem firmware upgrade binaries for the modem firmware" sounds a bit too much, maybe replace with something like "full binaries of the modem firmware" ?

Reply:
right, fixed